flink-runtime/src/test/java/org/apache/flink/runtime/io/network/api/CheckpointBarrierTest.java

```diff
@@ -22,37 +22,29 @@
 import org.apache.flink.core.memory.DataOutputSerializer;
 import org.apache.flink.runtime.checkpoint.CheckpointOptions;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-import static org.junit.Assert.fail;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /** Tests for the {@link CheckpointBarrier} type. */
-public class CheckpointBarrierTest {
+class CheckpointBarrierTest {
 
     /**
      * Test serialization of the checkpoint barrier. The checkpoint barrier does not support its own
      * serialization, in order to be immutable.
      */
     @Test
-    public void testSerialization() throws Exception {
+    void testSerialization() {
         long id = Integer.MAX_VALUE + 123123L;
         long timestamp = Integer.MAX_VALUE + 1228L;
 
         CheckpointOptions options = CheckpointOptions.forCheckpointWithDefaultLocation();
         CheckpointBarrier barrier = new CheckpointBarrier(id, timestamp, options);
 
-        try {
-            barrier.write(new DataOutputSerializer(1024));
-            fail("should throw an exception");
```

Comment:
Should we keep the hint message  by 
```
 assertThatThrownBy(() -> barrier.write(new DataOutputSerializer(1024))).as("should throw an exception")
                .isInstanceOf(UnsupportedOperationException.class);
```

?

Reply:
I don't think it adds any value. The `fail("should throw an exception")` is there simply to make sure, the exception is thrown